public/class-h5p-plugin.php

```diff
@@ -98,7 +98,7 @@ private function __construct() {
     add_action('init', array('H5P_Plugin', 'check_for_updates'), 1);
 
     // Add menu options to admin bar.
-    add_action('admin_bar_menu', array($this, 'admin_bar'));
+    add_action('admin_bar_menu', array($this, 'admin_bar'), 999);
```

Comment:
Wouldn't a smaller number be sufficient? This would probably put H5P at the bottom?